src/components/structures/MatrixChat.js

```diff
@@ -597,8 +597,8 @@ export default React.createClass({
                 break;
             case 'view_user_settings': {
                 const UserSettingsDialog = sdk.getComponent("dialogs.UserSettingsDialog");
-                Modal.createTrackedDialog('User settings', '', UserSettingsDialog, {}, 'mx_SettingsDialog',
-                    /*isPriority=*/false, /*isStatic=*/true);
+                Modal.createTrackedDialog('User settings', '', UserSettingsDialog, {},
+                    /*className*/null, /*isPriority=*/false, /*isStatic=*/true);
```

Comment:
Extremely minor nit: there should be an equal sign after the `className` to match the others. Same applies to `RoomViewStore`.

Reply:
oh yeah - did not spot that!